lib/db.js

```diff
@@ -66,6 +61,7 @@ function connect() {
 exports.addCallback = function(callback) {
 	if (db) {
 		if (isClosed === true) {
+			console.log('closed; connecting');
```

Comment:
I was just debugging an issue with tests, done in commit 9600516cc052617a69b5d550639623149246a49f.